### ISSUE TITLE
[4.0] Unsubstituted exeption and error texts

### ISSUE THREAD

**Issue by csbalazs** [joomla/joomla-cms#35588]
I get the following error:
![image](https://user-images.githubusercontent.com/1657924/133806677-670fe4e6-50e6-4c03-bdfb-ad4e324fac5f.png)

These files contain "burnt-in" text: 

/../libraries/src/Error/Renderer/CliRenderer.php

```
	public function render(\Throwable $error): string
	{
		$buffer = PHP_EOL . 'Error occurred: ' . $error->getMessage() . PHP_EOL . $this->getTrace($error);

		if ($prev = $error->getPrevious())
		{
			$buffer .= PHP_EOL . PHP_EOL . 'Previous Exception: ' . $prev->getMessage() . PHP_EOL . $this->getTrace($prev);
		}

		return $buffer;
```

/../libraries/vendor/symfony/error-handler/ErrorHandler.php

```
    public function handleException(\Throwable $exception)
    {
        $handlerException = null;

        if (!$exception instanceof FatalError) {
            self::$exitCode = 255;

            $type = ThrowableUtils::getSeverity($exception);
        } else {
            $type = $exception->getError()['type'];
        }

        if ($this->loggedErrors & $type) {
            if (false !== strpos($message = $exception->getMessage(), "@anonymous\0")) {
                $message = $this->parseAnonymousClass($message);
            }

            if ($exception instanceof FatalError) {
                $message = 'Fatal '.$message;
            } elseif ($exception instanceof \Error) {
                $message = 'Uncaught Error: '.$message;
            } elseif ($exception instanceof \ErrorException) {
                $message = 'Uncaught '.$message;
            } else {
                $message = 'Uncaught Exception: '.$message;
            }
```

**Comment by PhilETaylor** [comment]
`vendor/symfony/error-handler/ErrorHandler.php` is an upstream project. Joomla will not translate this.

Also in the Joomla command line console is in english only, and again not translated on purpose that that level of error handling.

The screenshot is unrelated to the CLI though... how do you generate this exception in the Joomla 4 Admin Console?

**Comment by csbalazs** [comment]
I got it under Wampserver (3.2.3.3) after a long inactivity. Later I tried to find the untranslated string in the source code, but I did not find it. If we cannot reproduce the error, then the thread can be closed.

**Comment by PhilETaylor** [comment]
"hozzaferes elutasitva" google translate has as Hungarian for "Rejected".

Checking the hu-HU language pack I see

```
 JGLOBAL_AUTH_ACCESS_DENIED="Hozzáférés elutasítva"
```

which is used through Joomla for generically denying access to a feature, mainly in Updates and installers, and more specifically in your case - on the Home Dashboard page when running the ajax calls to get the badge data for number of updates available etc... 

Without other users also having the issue, I would say that your session expired and that was the cause. 

I have attempted to replicate this but I failed to replicate it.
